Anchor.toml

```diff
@@ -2,28 +2,43 @@
 m2 = "M2mx93ekt1fmXSVkTrUL9xVFHkmME8HTUi5Cyc5aF7K"
 
 [programs.localnet]
-m2 = "M2mx93ekt1fmXSVkTrUL9xVFHkmME8HTUi5Cyc5aF7K"
+m2 = { address = "M2mx93ekt1fmXSVkTrUL9xVFHkmME8HTUi5Cyc5aF7K", idl = "./target/idl/m2.json" }
 
 [registry]
 url = "https://anchor.projectserum.com"
 
 [provider]
 cluster = "localnet"
-wallet = "~/.config/solana/id.json"
+wallet = "./devkey.json"
 
 [scripts]
-test = "cargo test"
-update_idl = "cp -r ./target/idl ./src && cp -r ./target/types ./src"
+test = "pnpm jest"
+copy_idl = "cp ./target/types/m2.ts ./src/services/m2.ts"
```

Comment:
probably want to revert this change back and update the IDL at https://github.com/magicoss/m2/blob/main/src/idl/m2.json

Reply:
Good catch